pages/community/precice-workshop-organizing.md

```diff
@@ -0,0 +1,122 @@
+
+The rooms for the training and the talks are typically different, even if the schedule is typically structured as a single track. The catering and the world café can typically use the same room and tables, but reusing the collaborative working room for the world café is also fine.
+
+### Define financing scheme
+
+Typically, the workshop is co-funded by participation fees. An important dependency for subsequent steps is which entity can collect and process such participation fees. This might be an association supporting scientific activities, an event management unit of a university, or an external company. The setup should allow to model an income to reinvest into maintaining preCICE.
+
+Multiple types of tickets are typically defined:
+
+- early -vs- late registration
+- full price -vs- discount for academia
+- training course included -vs- only main part of the workshop
+
+In addition, free registration is accounted for:
+
+- holders of a support license
+- invited speakers
+- organizers (if possible)
+- special cases
+
+Finally, account for a number of travel grants (at least for the invited speaker), if possible.
+
+### Set milestones
+
+Set clear responsible persons for each task, and track progress in a collaborative project board.
+
+Important dates to set:
+
+- "save the date" announcement: one year ahead
+- call for contributions (needs a form): open six months ahead. An early call mainly acts as advertisement and helps participants that need a visa application to travel.
+- open registration (needs a form): the earlier the better, and useful as a concrete action when advertising.
+- abstract submission deadline and contribution acceptance: the earlier the better, typically before the early-bird registration deadline. Aim for 3-4 months before the workshop, to facilitate the visa process. The contribution acceptance can be as short as one week after the deadline.
+- early-bird registration deadline: important for estimating the number of participants, driven by the catering requirements.
+- request catering offers: typically 3-4 months before the workshop.
+- reserve tables for the workshop dinner: typically 3-4 months before the workshop.
+- late registration deadline: typically 2-3 weeks before the workshop, to give an accurate number of participants to the caterer.
+
+## Registration infrastructure
+
+Two forms are mainly needed, and it is easier if they are decoupled: an abstract submission form, and a registration form.
+
+The abstract submission form should collect:
+
+- Kind of contribution (talk or poster, if offered)
+- Presenter
+- Authors (with the note: "preliminary list, can be modified later")
```

Comment:
```suggestion
- Authors
```
I would not put such a note. If needed any detail can be fixed later. But we should encourage complete submissions :)